e2eTests/e2eTestPrompts.md

```diff
@@ -39,6 +39,29 @@ This file contains prompts used for end-to-end testing to ensure each tool is in
 | azmcp-appconfig-kv-show | Show the content for the key <key_name> in App Configuration store <app_config_store_name> |
 | azmcp-appconfig-kv-unlock | Unlock the key <key_name> in App Configuration store <app_config_store_name> |
 
+## Azure Arc Operations
+
+| Tool Name | Test Prompt |
+|:----------|:----------|
+| azmcp-arc-validate-prerequisites-aksee-cluster | dispay prerequisites for installing AKS Edge Essentials cluster and prompt for next steps |
+| azmcp-arc-validate-prerequisites-aksee-cluster | Show me the prerequisites for AKS Edge Essentials cluster installation |
+| azmcp-arc-validate-system-requirements-hyperv | Validate system requirements and setup Hyper-V for AKS Edge Essentials. Use <path> for set up |
+| azmcp-arc-validate-system-requirements-hyperv | Check my system requirements for AKS Edge Essentials and configure Hyper-V if required. Use <path> for set up  |
+| azmcp-arc-validate-install-software-requirements | Validate and install software requirements for AKS Edge Essentials. Use <path> for set up  |
+| azmcp-arc-validate-install-software-requirements | Check and install required software components for AKS Edge. Use <path> for set up  |
+| azmcp-arc-remove-Aks-Edge-installation | Remove existing AKS Edge Essentials installation. Use <path> for set up  |
+| azmcp-arc-remove-Aks-Edge-installation | Clean up AKS Edge Essentials installation in from my system. Use <path> for set up |
```

Comment:
"Clean up AKS Edge Essentials installation in from my system" → "Clean up AKS Edge Essentials installation from my system" ?